end-end-tests/api-standards/resources/repos/2022-04-04/spec.yaml

```diff
@@ -1,5 +1,6 @@
 openapi: 3.0.3
 x-snyk-api-stability: experimental
+x-optic-url: https://app.useoptic.com/organizations/orgid/apis/apiId
```

Comment:
I've added a "fake" optic url here since these are just used for test cases, we don't need to upload anything but need them to be picked up by optic cloud